src/lib/core/CMakeLists.txt

```diff
@@ -47,6 +47,9 @@ set(core_sources
     tweaks.c
 )
 
+# Required for sqlInt.h.
+include_directories(${PROJECT_BINARY_DIR}/src/box/sql)
```

Comment:
Why do you need this?

Reply:
it was needed for including `stdInt.h`, removed